homeassistant/components/zwave_js/api.py

```diff
@@ -505,7 +508,10 @@ def async_cleanup() -> None:
             unsub()
 
     connection.subscriptions[msg["id"]] = async_cleanup
-    msg[DATA_UNSUBSCRIBE] = unsubs = [node.on("ready", forward_event)]
+    msg[DATA_UNSUBSCRIBE] = unsubs = [
```

Comment:
why store this in `msg` too ?

Reply:
so we can unsubscribe when an exception is raised in the exception handler decorator

Reply:
but why reuse the `msg` variable? That's the passed in message. Why not use a normal variable?

Reply:
the decorator wouldn't have access to a normal variable? unless I am missing something obvious